content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -119,10 +122,26 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
     exhibitionGuideId,
     exhibitionTitle,
     stopNumber,
-    totalStops,
+    allStops,
   } = props;
   const headerRef = useRef(null);
 
+  // We use the `shallow` prop with NextLinks to avoid doing an unnecessary
+  // `getServerSideProps` using the Previous/Next liks, because we already have
+  // all the data we need and can work it out client side
+  const router = useRouter();
+  const [stopNumberClientSide, setStopNumberClientSide] = useState(stopNumber);
+  const [currentStopClientSide, setCurrentStopClientSide] =
```

Comment:
NP: A comment to explain here _could_ be beneficial to future us/new joiners?
I'd also be tempted to revert the logic and have the SS stop number be identified as such and the `ClientSide` suffix be removed from the component for readability?

Reply:
Yeah I think that's preferable – will update thanks